packages/visual-editor/src/components/puck/atoms/button.tsx

```diff
@@ -71,60 +70,9 @@ export interface ButtonProps
 const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
   ({ className, variant, asChild = false, ...props }, ref) => {
     const Comp = asChild ? Slot : "button";
-    const [hasDarkBackground, setHasDarkBackground] = React.useState(false);
-
-    // handleRef determines if the active background color is light or dark and passes along the ref
-    const handleRef = React.useCallback((node: HTMLButtonElement | null) => {
```

Comment:
Nice that all of this can be removed. Is none of the logic inside necessary somewhere else now?